policyengine_uk/parameters/gov/hmrc/tax_free_childcare/minimum_weekly_hours.yaml

```
@@ -0,0 +1,10 @@
+description: Claimants must work sufficient hours to earn the equivalent of 16 hours per week at National Minimum Wage.
```

Ensure all labels and descriptions refer to the program and avoid values in them
```suggestion
description: HMRC limits tax-free childcare to benefit units in which each spouse earns at least the product of their minimum wage and this number of hours per week.
```